.buildkite/commands/build-android.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -eu
+
+echo "--- :npm: Install Node dependencies"
```

Comment:
If is "install" is intentional, that is completely fine and please disregard this. I just noted that we currently mix the usage of "install" and "set up." 
```suggestion
echo "--- :npm: Set up Node dependencies"
```